event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinMapper.java

```diff
@@ -54,9 +58,40 @@ public static DbCandidate toDbCandidate(CristinNviReport cristinNviReport) {
                    .applicable(true)
                    .createdDate(now)
                    .modifiedDate(now)
+                   .basePoints(extractBasePoints(cristinNviReport))
+                   .collaborationFactor(extractCollaborationFactor(cristinNviReport))
+                   .internationalCollaboration(isInternationalCollaboration(cristinNviReport))
                    .build();
     }
 
+    private static boolean isInternationalCollaboration(CristinNviReport cristinNviReport) {
+        return cristinNviReport.scientificResources().get(0).getCreators().stream()
+                   .map(ScientificPerson::getCollaborationFactor)
+                   .filter(Objects::nonNull)
+                   .findFirst()
+                   .map(INTERNATIONAL_COLLABORATION_FACTOR::equals)
+                   .orElse(false);
+    }
+
+    private static BigDecimal extractCollaborationFactor(CristinNviReport cristinNviReport) {
+        return cristinNviReport.scientificResources().get(0).getCreators().stream()
+                   .map(ScientificPerson::getCollaborationFactor)
+                   .filter(Objects::nonNull)
+                   .map(BigDecimal::new)
+                   .map(bigDecimal -> bigDecimal.setScale(1, RoundingMode.HALF_UP))
+                   .findFirst()
+                   .orElse(null);
+    }
+
+    private static BigDecimal extractBasePoints(CristinNviReport cristinNviReport) {
+        return cristinNviReport.scientificResources().get(0).getCreators().stream()
+                   .map(ScientificPerson::getPublicationTypeLevelPoints)
+                   .filter(Objects::nonNull)
+                   .map(BigDecimal::new)
+                   .map(bigDecimal -> bigDecimal.setScale(1, RoundingMode.HALF_UP))
```

Comment:
Extract scale and rounding mode to constant?

Reply:
Also, make sure no previous values for `publicationTypeLevelPoints` has more than one decimal if you set set scale to 0